### PR TITLE
Py310support

### PR DESCRIPTION
**Pull request by ritumd**
## Checklist before submitting

- [X] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes [#563](https://github.com/open-ce/open-ce/issues/563)

open-ce/open-ce#539

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: envs/arrow-env.yaml [hunk 1]
```diff
@@ -32,3 +32,5 @@ packages:
 {% endif %}
   - feedstock : grpc-cpp
   - feedstock : pyarrow
+
+git_tag_for_env: open-ce-v1.4.1
```
Comment (alexlang74) [inline envs/arrow-env.yaml:36]: shouldn't this be `v1.5.1` ?

Comment (ritumd) [reply envs/arrow-env.yaml:36]: yes @alexlang74. I will remove this and It's not added in our 1.5.1 build update.

Comment (alexlang74) [reply envs/arrow-env.yaml:36]: Thanks, @ritumd !

**Comment by ritumd** [comment]
PR is not needed for v1.5.1

**Pull request closed without merging**